packages/core/source/rule.ts

```diff
@@ -109,7 +109,7 @@ export default function parseRule(
 		}
 
 		// If the `par défaut` value is used, then the rule should be listed as a missingVariables
-		if (ruleValue['par défaut']) {
+		if (ruleValue['par défaut'] || ruleValue['par défaut'] === 0) {
```

Comment:
```suggestion
		if (ruleValue['par défaut'] != null) {
```

Tant qu'à faire autant gérer tous les cas (y compris les chaines vides par exemple)